src/pages/graphql/schema/b2b/purchase-order-rule/interfaces/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: PurchaseOrderApprovalRuleConditionInterface attributes and implementations
+edition: b2b
+---
+
+# PurchaseOrderApprovalRuleConditionInterface attributes and implementations
+
+`PurchaseOrderApprovalRuleConditionInterface` provides details about the approval rule conditions. It has the following implementations:
+
+*  [`PurchaseOrderApprovalRuleConditionAmount`](#purchaseorderapprovalruleconditionamount-attributes)
+*  [`PurchaseOrderApprovalRuleConditionQuantity`](#purchaseorderapprovalruleconditionquantity-attributes)
+
+## Attributes
+
+The `PurchaseOrderApprovalRuleConditionInterface` defines the following attributes.
+
+Attribute |  Data Type | Description
```

Comment:
Tables on the devsite don't have enough padding. Outside the scope of this PR, just noting it.